src/lib/annotations/BoxAnnotations.js

```diff
@@ -104,12 +113,19 @@ class BoxAnnotations {
 
         modifiedAnnotator = Object.assign({}, annotator);
 
-        // Filter out disabled annotation types
-        if (Array.isArray(viewerConfig.disabledTypes)) {
-            modifiedAnnotator.TYPE = modifiedAnnotator.TYPE.filter((type) => {
-                return !viewerConfig.disabledTypes.includes(type);
-            });
-        }
+        const enabledTypes = viewerConfig.enabledTypes || [...modifiedAnnotator.DEFAULT_ENABLED];
```

Comment:
Why do we need the spread operator and then an array? Isn't `modifiedAnnotator.DEFAULT_ENABLED` already an array?

Reply:
I want to make a copy of the array, not modify the original